hail_search/queries/base.py

```diff
@@ -66,7 +68,9 @@ class BaseHailTableQuery(object):
     TRANSCRIPTS_FIELD = None
     CORE_FIELDS = [XPOS]
     BASE_ANNOTATION_FIELDS = {
-        FAMILY_GUID_FIELD: lambda r: r.family_entries.filter(hl.is_defined).map(lambda entries: entries.first().familyGuid),
+        FAMILY_GUID_FIELD: lambda r: hl.array(hl.set(
+            r.family_entries.filter(hl.is_defined).map(lambda entries: entries.first().familyGuid))
+        ).filter(hl.is_defined),
```

Comment:
I also don't remember why I changed this so I will remove it..

Reply:
I'm not sure why this didn't happen in the existing flow, but family_entries can contain an entry that is [NA] (family has 1 sample and didn't pass checks so it will be filtered out) which is defined in hail world.

Reply:
A missing entry within a family can also be a valid represention that a given sample in the family was not called in the VCF for that particular variant. In that case it would be missing in the project/family table directly, regardless of inheirtance/quality filtering

Reply:
> I'm not sure why this didn't happen in the existing flow,

Can we confirm that this actually did happen in the existing flow? It seems unlikely that our inheritance filtering would let this through

Reply:
I removed the filter I added here and moved this to where I filter quality/inheritance and merge wes/wgs entries. 

I believe that the filter is necessary because in the case where there is no call for a family in wgs but a passing family in wes, we will keep the wgs call even though it's an array of None, and this needs to be explicitly handled when we merge wes/wgs.

Reply:
> the case where there is no call for a family in wgs but a passing family in wes

In this case we would want to keep the WES only, not keep the missing call for WGS